test/WorkOSTests/Services/SSO/SSOServiceTest.cs

```diff
@@ -321,5 +321,21 @@ public async void TestGetConnection()
                 JsonConvert.SerializeObject(this.mockConnection),
                 JsonConvert.SerializeObject(response));
         }
+
+        [Fact]
+        public async void TestDeleteConnection()
+        {
+            this.httpMock.MockResponse(
+                HttpMethod.Delete,
+                $"/connections/{this.mockConnection.Id}",
+                HttpStatusCode.OK,
```

Comment:
This should be a 204 No Content to match the API:

```suggestion
                HttpStatusCode.NoContent,
```

Reply:
HTTP 202 can also be returned by either operation and would imply that the instruction was accepted by the server, but not fully applied yet.

I can change the other tests to 204 instead if we prefer to do it that way.

Reply:
Changed to 204 [9d43662](https://github.com/workos/workos-dotnet/pull/110/commits/9d43662a80d7fb96f931415c16e2c9d0c497f630), can change the other tests to also return 204 no content.